### PR TITLE
#15331 add infos about deprecated classes to UPGRADE-3.0

### PR DESCRIPTION
**Pull request by vincentaubert**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #15331 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: UPGRADE-3.0.md [hunk 1]
```diff
@@ -272,6 +272,12 @@ UPGRADE FROM 2.x to 3.0
    ```php
    echo $form->getErrors(true, false);
    ```
+   
+   * The `Symfony\Component\Form\Extension\Core\ChoiceList\SimpleChoiceList` class has been removed in
+      favor of `Symfony\Component\Form\ChoiceList\ArrayChoiceList`.
```
Comment (xabbuh) [inline UPGRADE-3.0.md]: looks like you can remove one space at the beginning of the line

File: UPGRADE-3.0.md [hunk 2]
```diff
@@ -272,6 +272,12 @@ UPGRADE FROM 2.x to 3.0
...
+      
+   * The `Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList` class has been removed in
+      favor of `Symfony\Component\Form\ChoiceList\ArrayChoiceList`.
```
Comment (xabbuh) [inline UPGRADE-3.0.md]: same here

**Comment by xabbuh** [comment]
You should also add the deprecated `LayChoiceList` and `ObjectChoiceList` classes.

**Comment by vincentaubert** [comment]
fixed

**Comment by xabbuh** [comment]
:+1: 

Status: Reviewed

**Comment by fabpot** [comment]
Thank you @vincentaubert.

**Pull request merged**